docs/userGuide/syntax/variables.mbdf

```diff
@@ -159,7 +159,42 @@ In this case, all the variables in `title.md` are not accessible, as they are ov
 
 </box>
 
-### Defining variables with JSON
+### Importing variables from other external file formats
+
+You can also sources variables from external files using MarkBind's {%raw%}`{% ext varName = "filepathToFile/file.json" %}`{%endraw%} nunjucks extension.
```

Comment:
varName -> sourceName?

Reply:
its the variable name (similar to `{% set varName = expression %}`)

Reply:
to us, it's a variable. To the user, it's a file containing possibly multiple variables. Not sure if it can cause confusion in the user's mind.

Reply:
> it's a file containing possibly multiple variables

I've changed it here to be scoped under varName (so everything is accessed as `varName.xx.yy` / `varName[0]` / ...)

This allows json files that have an array as its root as well and should encourage better variable scoping. It should be more consistent with other file formats down the line that might not have top level "keys" too (e.g. csvs)

Agree its a little too brief / confusing though, I'll add in some more clarification and the json source file into the example :-)

Reply:
Makes sense.
Sorry, I forgot that this feature is only for json files which means each file can be considered a single root variable.

Reply:
updated 

hopefully clearer now : )

Reply:
Looks good.
Hope we can support csv files as well in the future.